### PR TITLE
Add travis tests

### PR DESCRIPTION
**Pull request by cpaillet**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+/site
+
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
+
+# Pyre type checker
+.pyre/
+
```
Comment (pierrecdn) [inline .gitignore:130]: lot of stuff here, where does it comes from?

Comment (pierrecdn) [reply .gitignore:130]: My bad, answered in the commit body.

Comment (cpaillet) [reply .gitignore:130]: it's on the commit message ->  https://github.com/github/gitignore/blob/master/Python.gitignore

**Review by pierrecdn — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (mougams) [reply .gitignore:130]: Looks like a typical gitignore file (Pipenv.lock -> recommanded)

File: pylama.ini [hunk 2]
```diff
@@ -24,4 +24,5 @@ max_line_length = 100
 # E1101: Module 'x' has no 'y' member, but source is unavailable.
 # R0903: Too few public methods
 # W1202: Use % formatting in logging functions and pass the % parameters as argument
-disable = C0111,C0114,C0116,C0115,E1101,R0903,W1202
+# W0511: TODO
```
Comment (dclaisse) [inline pylama.ini:27]: C0301?

File: pylama.ini [hunk 3]
```diff
@@ -24,4 +24,5 @@ max_line_length = 100
...
+disable = C0111,C0114,C0116,C0115,E1101,R0903,W1202,W0511,C0301
```
Comment (dclaisse) [inline pylama.ini:28]: OPT: re-order C0116,C0115 to C0115,C0116

**Pull request merged**